File: spring-web-java/src/main/java/spring/web/java/infrastructure/persistence/MemberJpaRepository.java
package spring.web.java.infrastructure.persistence;

import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;

import spring.web.java.domain.model.entity.Member;

public interface MemberJpaRepository extends JpaRepository<Member, Long> {

	Optional<Member> findByAccount(String account);

	boolean existsByAccount(String account);
}

File: spring-web-kotlin/src/main/kotlin/spring/web/kotlin/infrastructure/persistence/MemberJpaRepository.kt
package spring.web.kotlin.infrastructure.persistence

import org.springframework.data.jpa.repository.JpaRepository
import spring.web.kotlin.domain.model.entity.Member

interface MemberJpaRepository : JpaRepository<Member, Long> {

    fun findByAccount(account: String): Member?

    fun existsByAccount(account: String): Boolean
}